client/src/containers/filters/years-range/component.tsx

```diff
@@ -94,6 +92,10 @@ export const YearsRangeFilter: React.FC<YearsRangeFilterProps> = ({
   // Prevent display when not loaded
   if (!isLoaded) return null;
 
+  const handleChangeEndYear = ({ value }) => {
+    onChange?.({ startYear, endYear: Number(value) });
```

Comment:
Aren't we passing the options as `number` ? We shouldn't need to parse the value at this endpoint, it should be a `number` already.